ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -61,19 +65,19 @@
       tags: build_tools
     - role: adoptopenjdk_install  # JDK11 Build Bootstrap
       jdk_version: 10
-      when: ansible_distribution != "Alpine"
+      when: ansible_distribution != "Alpine" and ansible_distribution != "Solaris"
```

Comment:
Gut feel is still that these guards would be better inside the role, so that all the logic relating to the available versions is in one place, but I wouldn't block the PR on that basis at the moment.

Reply:
I'm currently working on the macOS playbook which will use this role. I'll make those changes over there